includes/admin/class-give-settings.php

```diff
@@ -543,6 +543,19 @@ public function give_settings( $active_tab ) {
 							'type' => 'file'
 						),
 						array(
+						    'id'      => 'email_logo_size',

```

Comment:
@ravinderk I ran into an issue where the image that one of my clients wanted to use was too large to look good in an email.   The image was over 500px wide and I needed to manually enter the path to a smaller version of it.